lib/web/mage/apply/main.js

```diff
@@ -13,6 +13,8 @@ define([
     var dataAttr = 'data-mage-init',
         nodeSelector = '[' + dataAttr + ']';
 
+    const idleCallback = requestIdleCallback || setTimeout;
```

Comment:
@rogerdz 
safer way would be: `const idleCallback = window.requestIdleCallback ?? setTimeout;` otherwise Safari throws errors, same goes for `lib/web/mage/bootstrap.js`

Reply:
thank, i missed test with safari